rotkehlchen/chain/ethereum/modules/uniswap/v3/decoder.py

```diff
@@ -415,7 +423,6 @@ def _maybe_decode_v3_deposit_or_withdrawal(
                 evm_inquirer=self.evm_inquirer,
                 encounter=TokenEncounterInfo(tx_hash=context.transaction.tx_hash),
             )
-            token_with_data = self.eth if token_with_data == A_WETH else token_with_data
```

Comment:
this line was making that the check was always against ETH instead of ETH/WETH. The uniswap pools always have WETH afaik and allow to withdraw/deposit ETH